### DISCUSSION TITLE
Host hostname

### DISCUSSION THREAD

**Discussion by ianschmitz** [lima-vm/lima#389] [Q&A] [answered]
I realize the title is confusing 😛 . Sorry if there's a better forum for questions. I didn't see GitHub discussions enabled on this repo.

Coming from docker desktop, the recommended pattern when wanting to refer to HTTP URLs on the macOS host was to use `host.docker.internal`. I'm wondering if there's something similar in the Lima world?

A concrete example is running a Postgres instance using [Postgres.app](https://postgresapp.com/) and wanting to connect to it from a container running in Lima.

**Comment by jandubois** [comment, marked as answer]
> Coming from docker desktop, the recommended pattern when wanting to refer to HTTP URLs on the macOS host was to use `host.docker.internal`. I'm wondering if there's something similar in the Lima world?

We define `host.lima.internal` to point to the host from inside the VM.

**Comment by jandubois** [comment]
> I didn't see GitHub discussions enabled on this repo.

@AkihiroSuda Should we enable discussions, so we can move questions out of Issues, which should only have actionable items?

**Reply by AkihiroSuda** [reply]
Done

**Comment by ianschmitz** [comment]
> We define `host.lima.internal` to point to the host from inside the VM.

Wonderful. I'll give that a try. I had looked through the docs and didn't find this (sorry if I missed!) - so may be good to document it? 

Cheers

**Comment by jandubois** [comment]
> I had looked through the docs

It is documented here:

https://github.com/lima-vm/lima/blob/5237bfe/docs/network.md#host-ip-19216852

**Comment by rfay** [comment]
A note though - `host.lima.internal` should be defined *inside* running containers. That's what docker desktop has always done, it provides that name resolution. People don't really want to get to the host just from inside the lima container, they want to get to the host from inside containers they're running.
